src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  background-color: #f4f4f4;
+  font-family: Roboto, Arial, sans-serif;
+
+  --accent-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo-link {
+  display: flex;
+}
+
+.nav-links {
+  list-style: none;
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). Instead, style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. Here, the `ul` tag is styled directly, which could cause unintended side effects if there are other `ul` elements in the project that shouldn't have the same style.